toponetx/__init__.py

```diff
@@ -18,13 +18,17 @@
 from .classes.reportviews import CellView, HyperEdgeView, SimplexView, NodeView
 
 
-from .datasets.mesh import stanford_bunny
+from .datasets.mesh import stanford_bunny, shrec_16
```

Comment:
These imports are not in alphabetic order, I wonder why flake8 did not see this...

Reply:
--> They should be in alphabetic order

Reply:
I think flake8 doesn't check that, but isort. And for some reason, isort ignores all \_\_init__.pys:

(from pyproject.toml)
```toml
[tool.isort]
multi_line_output = 3
include_trailing_comma = true
skip = [".gitignore", "__init__.py", ".venv"]
```